README.md

```diff
@@ -10,7 +10,7 @@ npm install -g autorest
 This AutoRest extension generates TypeScript code that is compatible with:
 ```
 "@azure/core-arm": "^1.0.0-preview.1",
-"@azure/core-http": "^1.0.0-preview.1"
```

Comment:
Maybe this should go in the core-http bump PR?  Not really a big deal since it's just a README comment.

Reply:
Good catch thanks!